__tests__/components/transaction-date.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react"
+import { render } from "@testing-library/react-native"
+import { TransactionDate } from "../../app/components/transaction-date"
+import {
+  AppConfiguration,
+  AppConfigurationContext,
+  defaultConfiguration,
+} from "../../app/context/app-configuration"
+import { GaloyGQL, translateUnknown } from "@galoymoney/client"
+import { createMock } from "ts-auto-mock"
+import moment from "moment"
+import i18n from "i18n-js"
+
+describe("Display the createdAt date for a transaction", () => {
+  it("Displays pending for a pending onchain transaction", () => {
+    const mockedTransaction = createMock<GaloyGQL.Transaction>({
+      status: "PENDING",
+      createdAt: new Date().getDate(),
+    })
+
+    const { queryAllByText } = render(
+      <AppConfigurationContext.Provider
+        value={{
+          appConfig: defaultConfiguration,
+          setAppConfig: (config: AppConfiguration) => {},
+        }}
+      >
+        <TransactionDate tx={mockedTransaction} />
```

Comment:
```TransactionDate``` component doesn't rely on AppConfigurationContext, so the provider is not necessary for these tests.

Reply:
Yeah I think I copied this pattern from another test and didn't consider that it wasn't needed.  Removed now.